Winium/Winium.StoreApps.Driver/EmulatorHelpers/Deployer.cs

```diff
@@ -155,6 +155,25 @@ public void SendFiles(Dictionary<string, string> files)
             }
         }
 
+        public void InstallDependencies(List<string> dependencies )
+        {
+            if (dependencies == null || !dependencies.Any())
+            {
+                return;
+            }
+
+            foreach (var dependency in dependencies)
+            {
+                InstallDependency(dependency);
+            }
+        }
+
+        public void InstallDependency(string path)
+        {
+            var appManifest = Utils.ReadAppManifestInfoFromPackage(path);
+            Utils.InstallApplication(this.deviceInfo, appManifest, DeploymentOptions.None, path);
+        }
```

Comment:
I suggest your second option ( returning `AppManifestInfo` ) because `AppManifest` has more information that might be useful at some point. But I'll add it to the pull request and then you can experiment and see if you want to change it or not